ext/mbstring/mbstring.c

```diff
@@ -1740,7 +1740,7 @@ PHP_FUNCTION(mb_language)
 	if (zend_parse_parameters(ZEND_NUM_ARGS() TSRMLS_CC, "|s", &name, &name_len) == FAILURE) {
 		return;
 	}
-	if (name == NULL) {
+	if (name_len == 0) {
```

Comment:
This doesn't look right to me. `mb_language("")` should trigger the "Unknown language" warning below, not return the current language.